apps/remix-ide/src/app/tabs/theme-module.js

```diff
@@ -37,7 +37,8 @@ export class ThemeModule extends Plugin {
     themes.map((theme) => {
       this.themes[theme.name.toLocaleLowerCase()] = {
        ...theme,
-        url: window.location.origin + window.location.pathname + theme.url
```

Comment:
is this change require? if remix is not the root level, this might not work...

Reply:
it just doesn't load the theme if pathname is not `/` . we should look for an alternative

Reply:
but the loading themes works at the moment in live, right?

Reply:
updated